ambry-cloud/src/main/java/com/github/ambry/cloud/azure/CosmosDataAccessor.java

```diff
@@ -72,15 +72,15 @@
   private static final String LIMIT_PARAM = "@limit";
   private static final String ACCOUNT_ID_PARAM = "@accountId";
   private static final String CONTAINER_ID_PARAM = "@containerId";
+  private static final String MAX_ENTRIES_PARAM = "@maxEntries";
```

Comment:
What's the difference between LIMIT and MAX_ENTRIES params?

Reply:
Removed MAX_ENTRIES.

Reply:
It doesn't look like it.